.github/workflows/nuget.yml

```diff
@@ -0,0 +1,47 @@
+name: Windows Nuget Build
+
+on:
+  push:
+    paths-ignore:
+      - 'CHANGES*'
+      - 'Doc/**'
+      - 'appveyor.yml'
+  pull_request:
+    branches: master
+    paths-ignore:
+      - 'CHANGES*'
+      - 'Doc/**'
+      - 'appveyor.yml'
+
+jobs:
+  build:
+
+    runs-on: windows-2019
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Install Dependencies
+      run: |
+          nuget install CMake-win64 -Version 3.15.5 -OutputDirectory C:\Tools\CMake
+          nuget install Bison -Version 3.7.4 -OutputDirectory C:\Tools\bison
+          nuget install PCRE2 -Version 10.39 -OutputDirectory C:\Tools\pcre2
+
+    - name: Build
+      shell: cmd
+      run: |
+          SET PATH=C:\Tools\CMake\CMake-win64.3.15.5\bin;C:\Tools\bison\Bison.3.7.4\bin;%PATH%
+          SET PCRE_ROOT=C:\Tools\pcre2\PCRE2.10.39.0
+          SET PCRE_PLATFORM=x64
+          cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_INSTALL_PREFIX="%CD:\=/%/install2" -DCMAKE_C_FLAGS="/DPCRE2_STATIC" ^
+          -DCMAKE_CXX_FLAGS="/DPCRE2_STATIC" -DPCRE2_INCLUDE_DIR=%PCRE_ROOT%/include -DPCRE2_LIBRARY=%PCRE_ROOT%/lib/pcre2-8-static.lib -S . -B build
+          cmake --build build --config Release --target install
```

Comment:
Best put into an Install section

Reply:
Will do

Reply:
Could you separate out the Build and Install into separate sections, otherwise looking good.

Reply:
I tried a [commit ](https://github.com/swig/swig/pull/2249/commits/730e49d379cad28dcbafe63c1d51b1a9bc68b9ee)with that, but the line `cmake --build build --config Release --target install` builds and installs with the same command. I can see if I can break in to 2 cmake commands?

Reply:
I'm a complete CMake newbie, but normally `cmake --build` without any `--target` should just build everything. And then `cmake --build xxx -t install` could be used for installing.

Reply:
Ok reading more there is a `cmake --install . --config Release` in newer versions of CMake that could be used. Will test now.